src/nvim/event/process.h

```diff
@@ -19,6 +19,7 @@ struct process {
   Loop *loop;
   void *data;
   int pid, status, refcount;
+  uint8_t exit_signal;  // store used signal when killing (used on Windows)
```

Comment:
"store" is redundant.

```suggestion
  uint8_t exit_signal;  // Signal used when killing (on Windows)
```